src/components/input-field/input-field.scss

```diff
@@ -135,6 +135,10 @@ input.mdc-text-field__input {
         top: 0;
         bottom: 0;
         margin: auto;
+
+        &:active {
+            transform: none; //Makes the "clear-all button" workable
```

Comment:
Would you mind to write "work" or "function", instead of "workable" ? (both in commit message and the comment here). 

The word workable may be confusing for the reader https://www.ldoceonline.com/dictionary/workable